conda/recipes/ucxx/meta.yaml

```diff
@@ -178,8 +178,8 @@ outputs:
         - cudatoolkit
         {% endif %}
         - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
-        - gtest
-        - gmock
+        - gtest {{ gtest_version }}
+        - gmock {{ gtest_version }}
```

Comment:
Would it be possible to change...

https://github.com/rapidsai/ucxx/blob/c3235d083bce472f7962c01a1a9c8867b1c3a034/conda/recipes/ucxx/conda_build_config.yaml#L26-L27

...to this?

```yaml
gmock:
  - ">=1.13.0"
gtest:
  - ">=1.13.0"
```

That way conda-build will pull in the versions for us (without needing templating) as is done with the other dependencies already in `conda_build_config.yaml`

Reply:
Interesting, I didn't know that conda would do this automatic pinning. We haven't been using this feature for other RAPIDS repos and I'm not sure if I like it, since it hides the fact that there is a version pinning from readers of `meta.yaml`. It's not very friendly to non-experts in conda build behavior, unfortunately. I'm somewhat inclined to do the opposite and change `cmake` to use a `cmake_version` variable in ucxx since that's how we pin `cmake` in other RAPIDS repos. What do you think @jakirkham?

Reply:
If you're confident that using pinnings like this in `conda_build_config.yaml` is best practice, we should file an issue and update the rest of RAPIDS. It just surprises me since I've been building recipes for some time without knowing about that feature.

Reply:
Updated in 4246795.